app/views/cart/show.html.erb

```diff
@@ -43,6 +45,9 @@
   <p>Total: <%= number_to_currency(cart.total) %></p>
   <p><%= link_to "Empty Cart", "/cart", method: :delete %></p>
   <p><%= link_to "Checkout", "/orders/new", method: :get %></p>
+  <% if session[:user_id].nil? %>
```

Comment:
Good and clean logic here!

Reply:
Is there an advantage to writing it this way as opposed to calling current_user?

Reply:
ah good catch Lito - I'd lean toward using `current_user` here for the sake of consistency re: how we're calling on this info.